jekyll/_cci2/configuration-reference.md

```diff
@@ -1126,7 +1093,7 @@ A job may have a `type` of `approval` indicating it must be manually approved be
 ```
 **Note:** The `hold` job name must not exist in the main configuration.
 
-###### **`filters`**
+##### **`filters`**
```

Comment:
this TOC level should stay six, not go to five (in the TOC we mimic that tabbing that should be used int he YAML)

Reply:
Fixed